Configuration/TypoScript/Page/setup.typoscript

```diff
@@ -8,3 +8,90 @@ page {
         ddbKitodoZeitungsportal = EXT:ddb_kitodo_zeitungsportal/Resources/Public/JavaScript/ddbKitodoZeitungsportal.js
     }
 }
+
+plugin.tx_dlf.useragent = DDB/Zeitungsportal
+
+plugin.tx_dlf_navigation {
+  pages = {$constant.configPid}
+  pageStep = 5
+  targetPid = {$constant.viewerPid}
+  templateFile = fileadmin/Templates/Navigation.html
```

Comment:
Please move this in the next step outside `fileadmin/` into this extension.

The line could be e.g.

```suggestion
  templateFile = EXT:ddb_kitodo_zeitungsportal/Resources/Private/Plugin/Kitodo/Navigation.html
```

Reply:
Same for all following template files.